Tests/ArcGISToolkitTests/CertificatePickerViewModelTests.swift

```diff
@@ -42,7 +42,7 @@ import XCTest
         // Another yield seems to be required to deal with timing when running the test
         // repeatedly.
         await Task.yield()
-        XCTAssertTrue(model.showCertificateImportError)
+        XCTAssertTrue(model.showCertificateError)
```

Comment:
This assertion seems to be failing.

Reply:
This passes for me. Which macOS, Xcode, and iOS versions are you testing?

Reply:
macOS Ventura 13.0.1
Xcode 14.1
iOS 15.5 & 16.1

Reply:
Test passes for me with this config. Does test fail for you with on `v.next` branch as well or just with branch of this PR? 

M1 Max
macOS Monterey 12.6.2
Xcode 14.1
iOS 16.1

Reply:
It passes okay for me on v.next.

Reply:
I ran it 1000x times and it failed twice.  So some timing involved.

Reply:
I see failure in run in repeat mode. Increasing sleep time on line 41 does not help. How to fix this?

Reply:
I'm not sure.  It seems rare and was probably already there, so you can put a bug in and we can try to address it later.

Reply:
Ok, https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/245